Macros/e3 Includes/e3_Classes_Bard.inc

```diff
@@ -125,6 +125,24 @@ SUB unpauseTwist
 	/delay 2 ${Me.Casting.ID}
 /return
 |--------------------------------------------------------------|
+#event toggleMez "<#1#> Mez #2#"
+#event toggleMez "#1# tells you, 'Mez #2#'"
+sub event_toggleMez(line, ChatSender, mezToggle)
+  /if (${Me.Class.ShortName.Equal[ENC]} || ${Me.Class.ShortName.Equal[BRD]}) {
+    /if (${Debug} || ${Debug_Ench}) /echo |- remove_RuneTarget ==>
+      /if (${mezToggle.Equal[on]}) {
+        /varset autoMez TRUE
+        /varset fixedMelody ${currentMelody}
```

Comment:
Just want to understand. Why does mezzing require a fixedMelody? Will changing melodies mess with autoMez?

Reply:
That seems fine